test/support/factory.ex

```diff
@@ -146,16 +146,24 @@ defmodule Wanda.Factory do
     }
   end
 
-  def execution_result_factory(attrs) do
-    execution_id = Map.get(attrs, :execution_id, Faker.UUID.v4())
-    group_id = Map.get(attrs, :group_id, Faker.UUID.v4())
+  def execution_result_factory do
+    %ExecutionResult{
+      execution_id: Faker.UUID.v4(),
+      group_id: Faker.UUID.v4(),
+      status: :running,
+      targets: [],
+      started_at: DateTime.utc_now()
+    }
+  end
 
+  def with_completed_status(
```

Comment:
Nice man!
But it is not used anywhere XD

Reply:
lol, if it's not too harm let's keep it, as a sample and for probable future usage